wgengine/monitor/polling.go

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2020 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//  +build !freebsd,!windows,!darwin android linux openbsd
```

Comment:
I can't convince myself this is correct.

The prior one you started one is tricky because of the fact that GOOS=android implies both the "android" and "linux" build tags. The file before was saying "only on Android, but not on non-Android Linux".

You added "linux" and "openbsd" here, but those are no-ops... they should be covered by the first part: `!freebsd,!windows,!darwin`.... Linux and OpenBSD are none of those three.

I think you just want:

```
// +build !freebsd,!windows,!darwin
```

(they have their own implementations)

Reply:
Done.